src/benchmarks/gc/src/exec/run_tests.py

```diff
@@ -119,23 +160,34 @@ def _run_all_benchmarks(
     skip_where_exists: bool,
     max_iterations: Optional[int],
     out_dir: Path,
+    run_errors: List[BenchmarkErrorInfo]
 ) -> None:
     default_env = check_env()
     for t in iter_tests_to_run(bench, get_this_machine(), max_iterations, out_dir):
         now = datetime.now().strftime("%H:%M:%S")
-        print(f"{now} Running {t.out.out_path_base.name}")
-        if not (skip_where_exists and t.out.exists()):
-            run_single_test(
-                built,
-                SingleTest(
-                    test=t.test,
-                    coreclr=built.coreclrs[t.coreclr_name],
-                    test_exe=_get_path(built, t.bench_file.paths, t.benchmark.get_executable),
-                    options=t.bench_file.options,
-                    default_env=default_env,
-                ),
-                t.out,
+
+        try:
+            print(f"{now} Running {t.out.out_path_base.name}")
+            if not (skip_where_exists and t.out.exists()):
+                run_single_test(
+                    built,
+                    SingleTest(
+                        test=t.test,
+                        coreclr=built.coreclrs[t.coreclr_name],
+                        test_exe=_get_path(built, t.bench_file.paths, t.benchmark.get_executable),
+                        options=t.bench_file.options,
+                        default_env=default_env,
+                    ),
+                    t.out,
+                )
+        except Exception:
+            _, exception_message, exception_trace = exc_info()
+            run_errors.append(
+                BenchmarkErrorInfo(t.benchmark_name,
```

Comment:
this gives the benchmark name but you could have different configs the same benchmark gets run with. can we also give the configs? the purpose of providing such errors is to help whoever investigates the error to construct the failure scenario and rerun it.

Reply:
I'm so used to running one configuration I forgot about that functionality. Will add it to the summary messages as well.

Reply:
the coreclr build is also parameter that can vary, from the doc:
"The test runner uses all combinations of coreclrs ⨯ configs ⨯ benchmarks."